tests/testthat/_snaps/telemetry.md

```diff
@@ -0,0 +1,135 @@
+# telemetry and arrange()
+
+    Code
+      tibble(a = 1:3, b = 4:6) %>% as_duckplyr_df() %>% arrange(a, .by_group = TRUE)
+    Condition
+      Error in `rel_try()`:
+      ! arrange: {"message":"Error in arrange","name":"arrange","x":{"...1":"integer","...2":"integer"},"args":{"dots":["...1"],".by_group":true}}
+
+# telemetry and count()
+
+    Code
+      tibble(a = 1:3, b = 4:6) %>% as_duckplyr_df() %>% count(a, wt = b, sort = TRUE,
+        name = "nn", .drop = FALSE)
+    Condition
+      Error in `rel_try()`:
+      ! count: {"message":"Error in count","name":"count","x":{"...1":"integer","...2":"integer"},"args":{"dots":{"1":"...1"},"wt":"...2","sort":true,"name":"nn",".drop":false}}
+
+# telemetry and distinct()
+
+    Code
+      tibble(a = 1:3, b = 4:6) %>% as_duckplyr_df() %>% distinct(a, b, .keep_all = TRUE)
+    Condition
+      Error in `rel_try()`:
+      ! distinct: {"message":"Error in distinct","name":"distinct","x":{"...1":"integer","...2":"integer"},"args":{"dots":{"a":"...1","b":"...2"},".keep_all":true}}
+
+# telemetry and filter()
+
+    Code
+      tibble(a = 1:3, b = 4:6) %>% as_duckplyr_df() %>% filter(a > 1, .by = b)
+    Condition
+      Error in `rel_try()`:
+      ! filter: {"message":"Error in filter","name":"filter","x":{"...1":"integer","...2":"integer"},"args":{"dots":{"1":"...1 > \"Don't know how to scrub numeric\""},"by":"...2","preserve":false}}
+
+---
+
+    Code
+      tibble(a = 1:3, b = 4:6) %>% as_duckplyr_df() %>% filter(a > 1, .preserve = TRUE)
+    Condition
+      Error in `rel_try()`:
+      ! filter: {"message":"Error in filter","name":"filter","x":{"...1":"integer","...2":"integer"},"args":{"dots":{"1":"...1 > \"Don't know how to scrub numeric\""},"by":"\"Don't know how to scrub NULL\"","preserve":true}}
```

Comment:
Just checking, you want to see "dont know how to scrub numeric" in these cases?

Reply:
Good question. I guess it's fine to use `"<character>"` for strings, and to pass through other types?